<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewMember</class>
 <widget class="QDialog" name="ViewMember">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>495</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>357</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>View Member</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(218, 218, 218);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>25</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <property name="bottomMargin">
    <number>25</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(213, 229, 246);</string>
     </property>
     <property name="title">
      <string>View Member</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_first">
            <property name="styleSheet">
             <string notr="true">color: rgb(128, 128, 128);</string>
            </property>
            <property name="text">
             <string>First Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_first">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_last">
            <property name="styleSheet">
             <string notr="true">color: rgb(128, 128, 128);</string>
            </property>
            <property name="text">
             <string>Last Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_last">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_email">
            <property name="styleSheet">
             <string notr="true">color: rgb(128, 128, 128);</string>
            </property>
            <property name="text">
             <string>Email Address:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_email">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_phone">
            <property name="styleSheet">
             <string notr="true">color: rgb(128, 128, 128);</string>
            </property>
            <property name="text">
             <string>Phone Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_phone">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_home_1">
            <property name="styleSheet">
             <string notr="true">color: rgb(128, 128, 128);</string>
            </property>
            <property name="text">
             <string>Home Address Line 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_home_1">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_home_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(128, 128, 128);</string>
            </property>
            <property name="text">
             <string>Home Address Line 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_home_2">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255);</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>3</number>
          </property>
          <property name="bottomMargin">
           <number>3</number>
          </property>
          <item>
           <widget class="QLabel" name="label_code">
            <property name="styleSheet">
             <string notr="true">color: rgb(128, 128, 128);</string>
            </property>
            <property name="text">
             <string>Post Code:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="line_code">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 255, 255)</string>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
